x-pack/plugins/observability/public/pages/home/section.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { usePluginContext } from '../../hooks/use_plugin_context';
+
+export interface ISection {
+  id: string;
+  title: string;
+  icon: string;
+  description: string;
+  href?: string;
+  target?: '_blank';
+}
+
+const Link = styled(EuiButtonEmpty)`
+  height: auto;
+  &:focus {
+    background-color: transparent;
+  }
+  &:hover {
+    .title {
+      text-decoration: underline;
+    }
+  }
+`;
```

Comment:
It would be better if we didn't overwrite the eui styles.

Reply:
This is the same as the `color` since the anchor tag automatically adds `text-decoration: underline` in the entire container, and we only want to underline the title not the description I needed to do this customization.

Reply:
Removed after changing to EuiCard component.